### DISCUSSION TITLE
Is there a way to install a Plugin by default

### DISCUSSION THREAD

**Discussion by ncecere** [danny-avila/LibreChat#3310] [Q&A]
I was wondering if there was a way to install a plugin by default for all users?  Like I wanted to make sure all users had "calculator" and "dall-e-3" install for them in the plugin system

**Comment by maxesse** [comment]
I think the more "modern" way to do this, would be to create an assistant which you make available to your users, you add those plugins as tools to the assistant, and in the instructions you tell the assistant how to use those available tools, like ChatGPT does.
